mappings/net/minecraft/command/CommandSource.mapping

```diff
@@ -5,13 +5,27 @@ CLASS net/minecraft/unmapped/C_sasfholk net/minecraft/command/CommandSource
 		ARG 0 candidates
 		ARG 1 builder
 	METHOD m_btzrwcvs getBlockPositionSuggestions ()Ljava/util/Collection;
+	METHOD m_drfhpdsu (Lnet/minecraft/unmapped/C_ncpywfca;)Lnet/minecraft/unmapped/C_ncpywfca;
+		ARG 0 identifier
 	METHOD m_edjrtbpe suggestFromIdentifier (Ljava/lang/Iterable;Lcom/mojang/brigadier/suggestion/SuggestionsBuilder;Ljava/util/function/Function;Ljava/util/function/Function;)Ljava/util/concurrent/CompletableFuture;
 		ARG 0 candidates
 		ARG 1 builder
 		ARG 2 identifier
 		ARG 3 tooltip
 	METHOD m_gmcggpep suggestMatching (Ljava/util/stream/Stream;Lcom/mojang/brigadier/suggestion/SuggestionsBuilder;)Ljava/util/concurrent/CompletableFuture;
+	METHOD m_hfcwokxp matchesTrimmedStr (Ljava/lang/String;Ljava/lang/String;)Z
+		COMMENT Tests if the trimmed string matches the original, regardless of trailing underscores
+		COMMENT <p> {@code matchesSubStr("ab", "abaa") == true}
+		COMMENT <p> {@code matchesSubStr("aa", "aa_") == true}
+		COMMENT <p> {@code matchesSubStr("aa:ab", "aa:aa") == false}
+		COMMENT <p> {@code matchesSubStr("_aa", "aa") == false}
```

Comment:
The method name doesn't match the examples

Reply:
examples?

Reply:
examples use `matchesSubStr`, whereas the method is actually `matchesTrimmedStr`

Reply:
oh ok